AIPscan/Aggregator/database_helpers.py

```diff
@@ -352,6 +365,8 @@ def create_file_object(file_type, fs_entry, aip_id):
     if file_type == FileType.preservation:
         _add_normalization_date(new_file.id)
 
+    _add_premis_object_xml(fs_entry, new_file.id)
+
```

Comment:
Maybe also add a return None here. I think you had the right idea in _add_premis_object_xml function to put a return statement. Even if it is implicit it is easier to read.